src/reports.coffee

```diff
@@ -152,44 +185,13 @@ plainTemplate = (report) ->
             "
   text
 
-htmlTemplate = (report) ->
-  text = "
-    <html>
-    <head></head>
-    <body>
-    <h1>#{report.type} OpenHIM Transactions Summary</h1>
-    <div>
-    <p>on the OpenHIM instance running on <b>#{config.alerts.himInstance}</b>:</p>
-    <p><span>Generated on: #{new Date().toString()}</span></p>
-    <table>
-    <tr>
-      <th>Channel Name</th>
-      <th>Channel Load</th>
-      <th>Ave response time</th>
-      <th>Failed</th>
-      <th>Successful</th>
-      <th>Processing</th>
-      <th>Completed</th>
-      <th>Completed with errors</th>
-    </tr>
-        "
-  for data in report.data
-    do (data) ->
-      text += "<tr><td><i>#{data.channel.name}</i></td>"
-      text += "<td> #{ if data.data[0]?.load? then data.data[0].load else 0 } transactions </td>"
-      text += "<td> #{ if data.data[0]?.avgResp? then data.data[0].avgResp else 0 } </td>"
-      text += "<td> #{ if data.statusData[0]?.failed? then data.statusData[0].failed else 0 }  </td>"
-      text += "<td> #{ if data.statusData[0]?.successful? then data.statusData[0].successful else 0 }  </td>"
-      text += "<td> #{ if data.statusData[0]?.processing? then data.statusData[0].processing else 0 }  </td>"
-      text += "<td> #{ if data.statusData[0]?.completed? then data.statusData[0].completed else 0 }  </td>"
-      text += "<td> #{ if data.statusData[0]?.completedWErrors? then data.statusData[0].completedWErrors else 0 } </td></tr>"
-  text += "
-    </table>
-    </div>
-    </body>
-    </html>
-    "
-  text
+renderTemplate = (templateName, templateData, callback) ->
+  templateDir = path.join(__dirname, '../src/templates', templateName)
```

Comment:
There are some issues with linking to files when the openhim is used as an npm package, so it would be best to use our appRoot global instead of `__dirname` for consistency: https://github.com/jembi/openhim-core-js/blob/master/src/server.coffee#L6